src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -35,17 +34,19 @@ class ConsumeMessagesCommand extends Command
 {
     protected static $defaultName = 'messenger:consume-messages';
 
-    private $bus;
+    private $busLocator;
     private $receiverLocator;
     private $logger;
     private $receiverNames;
+    private $busNames;
 
-    public function __construct(MessageBusInterface $bus, ContainerInterface $receiverLocator, LoggerInterface $logger = null, array $receiverNames = array())
+    public function __construct(ContainerInterface $busLocator, ContainerInterface $receiverLocator, LoggerInterface $logger = null, array $receiverNames = array(), array $busNames = array())
```

Comment:
This is a BC break. Can you update the UPGRADE file?

Reply:
@sroze Can you add a note in the CHANGELOG?